applications/rag/frontend/variables.tf

```diff
@@ -53,6 +53,24 @@ variable "db_secret_name" {
   description = "CloudSQL user credentials"
 }
 
+variable "base_image" {
+  type        = string
+  description = "Base image for the application"
+  default     = "us-central1-docker.pkg.dev/ai-on-gke/rag-on-gke/frontend@sha256:d65b538742ee29826ee629cfe05c0008e7c09ce5357ddc08ea2eaf3fd6cefe4b"
+}
+
+variable "local_image" {
+  type        = string
+  description = "Built off of the base image from local cloudbuild.yaml"
+  default     = "us-central1-docker.pkg.dev/ai-on-gke/rag-on-gke/amundson-frontend@sha256:ae19e033e5fd69315d9121bed0ae2cc5ae0e0f52615bf394325660fbe762c006"
```

Comment:
should we call it something else? probably don't want your ldap here?

Reply:
actually do we even need to show 2 images? why not just have 1 image with chat history built in? is it to demonstrate how to swap out images (which maybe we could just have a section in the README for?)

Reply:
Removing as this logic will be moved to another PR.